tools/amd_build/patches/a_aten_src_ATen_cuda_CUDAHalf.cu.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/aten/src/ATen/cuda/CUDAHalf.cu b/aten/src/ATen/cuda/CUDAHalf.cu
+index dbb18fc3f..5c4754f58 100644
+--- a/aten/src/ATen/cuda/CUDAHalf.cu
++++ b/aten/src/ATen/cuda/CUDAHalf.cu
+@@ -4,6 +4,7 @@
+ #include <cuda.h>
+ #include <cuda_runtime.h>
+ #include <cuda_fp16.h>
++#if !defined(__HIP_PLATFORM_HCC__)
```

Comment:
If this file is completely disabled in the HIP build, then maybe it's just simpler to exclude it from the build?